### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by jeetbhalani15**
image

### CODE REVIEW

**Review by Jay-Gadhiya — APPROVED** [review]

File: components/Buttons/button.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
+.btn{
+
```
Comment (ishitakeshawani) [inline components/Buttons/button.css:2]: extra space here

File: components/Lists/list.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    padding-inline-start: 0;
+}
+.list-pd{
+    border: 1px solid white;
+    padding: 1rem 5rem;
+}
+.styled-list{
+    margin-bottom: 1rem;
+}
+.bg{
```
Comment (ishitakeshawani) [inline components/Lists/list.css:12]: please give the proper classname. The name of a class is bg but it also includes height and width.

File: components/Lists/list.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    background-color: black;
+    color: white;
+    width: 300px;
+    height: 460px;
```
Comment (ishitakeshawani) [inline components/Lists/list.css:16]: try to use rem.

File: components/cards/card.css [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+    width: 32rem;
+    margin: 4rem;
+    border: var(--card-text);
+    border-radius: 8px;
+    background-color: var(--card-bg-color);
+    overflow: hidden;
+    box-shadow: 2px 2px 6px 3px darkgrey;
+}
+
+img{
```
Comment (ishitakeshawani) [inline components/cards/card.css:28]: class name and properties do not match.
give a meaningful name.

**Review by ishitakeshawani — CHANGES_REQUESTED** [review]
Left some comments.

**Comment by aurobindoGupta** [comment]
preferably reduce the size of pr, I guess this is because u have not merged the previous components. Good work!

File: components/Lists/list.css (hunk 2 above)
Comment (jeetbhalani15) [reply components/Lists/list.css:12]: ok done

File: components/Lists/list.css (hunk 3 above)
Comment (jeetbhalani15) [reply components/Lists/list.css:16]: ok

File: components/Buttons/button.css (hunk 1 above)
Comment (jeetbhalani15) [reply components/Buttons/button.css:2]: ok done

File: components/cards/card.css (hunk 4 above)
Comment (jeetbhalani15) [reply components/cards/card.css:28]: ohk ig its for image height and width ok i change it

**Pull request merged**